R-proj/src/vol_R.cpp

```diff
@@ -0,0 +1,111 @@
+// [[Rcpp::depends(BH)]]
+
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 20012-2018 Vissarion Fisikopoulos
+// Copyright (c) 2018 Apostolos Chalkis
+
+//Contributed and/or modified by Apostolos Chalkis, as part of Google Summer of Code 2018 program.
+#include <Rcpp.h>
+#include <RcppEigen.h>
+#include "../../include/volume.h"
```

Comment:
avoid includes with relative positions like "../../path" etc

Reply:
Numbertype and definitions of var, Point, random generator and kernel have to be done first. So they are below these declarations.

Reply:
linking improved. All the included and external files added to makevars and to cmakeLists.txt